app/views/runs/index.slim

```diff
@@ -27,7 +27,7 @@
     .row.mx-2
       .col-md-6.mb-3: .card.shadow
         = render partial: 'shared/race_table', locals: { \
-          races: current_user.races.finished.order(created_at: :desc), \
+          races: current_user.races.with_ends.finished.order(ended_at: :desc), \
```

Comment:
Do you think it would be worth it to move this sorting to inside the partial, so we don't have to repeat it every time we render the partial?

In fact on that note, should we also move the distinction between `races` and `active_races` inside the partial, and just pass in one `races` ActiveRecord::Relation that the partial then splits up using `.finished` and `.unfinished`?

Reply:
So I tried tinkering around with this a bunch, and was not able to get things working very well.  If you move just the `order(ended_at: :desc)` to the parial then you start getting column doesn't exist errors, if you move both `.with_ends.order(ended_at: :desc)` to the partial, then you start getting errors about the group by clause.

I also spent some time tinkering around with just passing in one race object, but I wasn't able to get the behavior the same as it was now, specifically on the active races page.  Maybe we can revisit this in the future though.

Reply:
Okay sounds good, thanks for explaining!